server/queue/queue-jms/src/test/java/org/apache/james/queue/jms/JMSMailQueueTest.java

```diff
@@ -37,17 +36,24 @@
 import org.junit.jupiter.api.extension.ExtendWith;
 
 @ExtendWith(BrokerExtension.class)
-public class JMSMailQueueTest implements DelayedManageableMailQueueContract, PriorityManageableMailQueueContract, DelayedPriorityMailQueueContract {
+public class JMSMailQueueTest implements DelayedManageableMailQueueContract, PriorityManageableMailQueueContract, DelayedPriorityMailQueueContract,
+    JMSMailQueueMetricContract {
 
-    private JMSMailQueue mailQueue;
+    JMSMailQueue mailQueue;
```

Comment:
What are these changes needed for?

Reply:
I need to access it in my test Extension